pkg/services/control/server/evacuate.go

```diff
@@ -39,3 +49,62 @@ func (s *Server) EvacuateShard(_ context.Context, req *control.EvacuateShardRequ
 	}
 	return resp, nil
 }
+
+func (s *Server) replicate(addr oid.Address, obj *objectSDK.Object) error {
+	nm, err := s.netMapSrc.GetNetMap(0)
+	if err != nil {
+		return err
+	}
+
+	cid, ok := obj.ContainerID()
+	if !ok {
+		// Container ID is expected to be here. If for some reason it is not,
+		// we log error and return nil to prevent situations
```

Comment:
log or not log?

also, could it be the first check before any network interaction?

Reply:
I don't think we need to complicate this, such errors should be caught in other places.
Moved network map fetching.

Reply:
what about "log"? mb i get it wrong, what exactly logging that comment is telling about? general? not logging that err here?

Reply:
Removed "log", the purpose of this comment is to justify returning `nil` even though missing CID is an error.